### PR TITLE
added sections to schema

### PR DESCRIPTION
**Pull request by dantavori**
## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-8295

## Description
added support for inputsSections and outputsSections in the playbook yml.

### CODE REVIEW

File: demisto_sdk/commands/common/schemas/playbook.yml [hunk 1]
```diff
@@ -311,6 +319,23 @@ schema;input_schema:
...
 
+# playbook->inputSections
+schema;inputsSection_schema:
+  type: map
+  mapping:
+    name:
+      type: str
+      required: true
+    id:
+      type: str
```
Comment (ShahafBenYakir) [inline demisto_sdk/commands/common/schemas/playbook.yml:330]: This is optional I understand? 
I think we need to make sure the ID will not be exported when exporting the playbook yml

File: demisto_sdk/commands/common/tests/playbook_test.py [hunk 2]
```diff
@@ -580,6 +580,12 @@ class TestPlaybookValidator:
         (INVALID_PLAYBOOK_INPUTS_USE, False, True),
     ]
 
+    def test_valid_schema(self, playbook):
```
Comment (ShahafBenYakir) [inline demisto_sdk/commands/common/tests/playbook_test.py:583]: Not sure I understand what this checks that was not checked so far?

**Review by ShahafBenYakir — APPROVED** [review]
Before merging, let's open a PR in content (having this SDK branch in the poetry) with this changes to some playbook and see that build passes

File: demisto_sdk/commands/common/tests/playbook_test.py (hunk 2 above)
Comment (dantavori) [reply demisto_sdk/commands/common/tests/playbook_test.py:583]: Yes there's a bit of a mess... I searched for `is_valid_scheme` in the repo and here's what we have:
* In `structure_test.py`:
   * For playbooks there's only `test_scheme_validation_playbook` which does not use TestSuite
   * There are 9 schema tests (with TestSuite) for incident/indicator fields and reputation
   * There are schema tests (with TestSuite) for all **XSIAM** content types
* In different test files, there are 3 tests for generic fields, and 1 test for layout rule

Except for these there are no tests for schemas at all.

File: demisto_sdk/commands/common/schemas/playbook.yml (hunk 1 above)
Comment (dantavori) [reply demisto_sdk/commands/common/schemas/playbook.yml:330]: It shouldn't be exported from XSOAR when we download it, I think we can remove this, but I'll ensure.

File: demisto_sdk/commands/common/tests/playbook_test.py (hunk 2 above)
Comment (ShahafBenYakir) [reply demisto_sdk/commands/common/tests/playbook_test.py:583]: Meaning we were not validating schema for playbooks?

Comment (dantavori) [reply demisto_sdk/commands/common/tests/playbook_test.py:583]: we validate in `test_scheme_validation_playbook`

**Pull request merged**